src/Group.php

```diff
@@ -87,6 +88,24 @@ public function withDispatcher(MiddlewareDispatcher $dispatcher): GroupInterface
         return $group;
     }
 
+    public function withAutoOptions(...$middlewares): GroupInterface
```

Comment:
Need phpdoc.

Reply:
Maybe it is worth naming it `withAutoOptionsMiddlewares()`. It is a bit longer but makes user to wonder less about the method purpose.